camunda-formio-tasklist-vue/src/styles/camundaFormIOTasklist.scss

```diff
@@ -572,7 +572,7 @@
 
   .cft-assignee-change-box .assignee-align .vs__selected {
     white-space: nowrap;
-    width: 100%;
+    width: 150px;
```

Comment:
We avoided fixed length and moved to % values earlier. @ann-aot can add something on why we did that then?

Reply:
the issue with the length of the text . we have options in side the select tag. so the options will over ride the select tag. i will fix this once again